### DISCUSSION TITLE
MVP II

### DISCUSSION THREAD

**Discussion by bellomuboye** [bellomuboye/make-we-yarn#4] [General]
Notifications when tagged in a room
Password Recovery
Delete profile
Multi Rooms
Role-based authorization in channels/workspace
Type of Messages
File Sharing
Notifications when tagged
Multi Rooms
Authorization in rooms/workspace
Type of Messages
Message Scheduler
File Sharing
Voice Calls

See full bucket here #2

**Comment by fmoliveira** [comment]
You did a great job of moving important but non-essential tasks to a second version! These will probably be split into more milestones, but we don't need to worry about it at the moment - the starting tasks from #3 were very well picked!
